### PR TITLE
Update prow config to add include main branch

### PR DESCRIPTION
**Pull request by Karthik-K-N**

### CODE REVIEW

File: core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml [hunk 1]
```diff
@@ -1,6 +1,23 @@
 tide:
   queries:
-  - labels:
+  - includedBranches:
+    - main
+    labels:
+    - approved
+    - lgtm
```
Comment (Prashanth684) [inline core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml:7]: add `bugzilla/valid-bug` here

File: core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+    - do-not-merge/work-in-progress
+    - needs-rebase
+    repos:
+    - openshift/machine-api-provider-powervs
+  - includedBranches:
+    - openshift-4.11
+    - release-4.11
+    labels:
     - approved
     - lgtm
```
Comment (Prashanth684) [inline core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml:22]: add `bugzilla/valid-bug`  and `staff-eng-approved` here in line with other repos

File: core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml (hunk 1 above)
Comment (Karthik-K-N) [reply core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml:8]: Added it

File: core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml (hunk 2 above)
Comment (Karthik-K-N) [reply core-services/prow/02_config/openshift/machine-api-provider-powervs/_prowconfig.yaml:25]: Added it

**Pull request merged**